### ISSUE TITLE
Draw a selection preview around windows

### ISSUE THREAD

**Issue by 9ary** [neXromancers/hacksaw#8]
Since a single click will return the clicked window's geometry, it would be nice to draw a rectangle around it like slop.

**Comment by tnjd** [comment]
Would the rectangle have the same style as the guides?

**Comment by JuanPotato** [comment]
@expectocode I wanted to reply to this to see what would happen. Test message please ignore

**Comment by 9ary** [comment]
No, it should use the style of the selection. With slop, I had my selection thickness the same as my border width for this exact reason.
